Sources/Commands/SwiftTool.swift

```diff
@@ -25,6 +25,7 @@ import SPMBuildCore
 import Build
 import XCBuildSupport
 import Workspace
+import Basics
```

Comment:
This is a typo, right? I don't think there is a `Basics` module?

Reply:
This is not a typo. It's used for the `FileSystem` extensions which give us `var dotSwiftPM: AbsolutePath` and `var swiftPMCacheDirectory: AbsolutePath` in FileSystem+Extensions.swift.
See Package.swift:106 for the module definition.

Reply:
@neonichu allot happened while you were away :D

Reply:
The smoke tests also seem to be unaware of this module.

Reply:
ugh we missed adding it in the CMake setup

Reply:
https://github.com/apple/swift-package-manager/pull/3064